File: payroll-api/src/main/java/com/company/payroll/model/StaffSalary.java
package com.company.payroll.model;

import java.math.BigDecimal;
import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
public class StaffSalary {
    private Integer sId;
    private BigDecimal monthlySalary;
    private BigDecimal annualSalary;
    private LocalDate dateUpdate;
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/StaffSalary.kt
package com.company.payroll.model

import com.company.payroll.annotation.NoArg
import lombok.AllArgsConstructor
import lombok.NoArgsConstructor
import lombok.RequiredArgsConstructor
import java.math.BigDecimal
import java.time.LocalDate

@NoArg
data class StaffSalary(
    var sId: Int,
    var monthlySalary: BigDecimal,
    var annualSalary: BigDecimal,
    var dateUpdate: LocalDate?
)
